docs/multicluster/quick-start.md

```diff
@@ -52,10 +52,11 @@ to deploy Antrea Multi-cluster and set up the example ClusterSet using `antctl`.
 A [further section](#steps-with-yaml-manifests) will describe the steps to
 achieve the same using YAML manifests.
 
-To execute any command in this section, `antctl` needs access to the target
-cluster's API server, and it needs a kubeconfig file for that. Please refer to
-the [`antctl` Multi-cluster manual](antctl.md) to learn more about the
-kubeconfig file configuration, and the `antctl` Multi-cluster commands.
+Please refer to the [`antctl` installation guide](../antctl.md#installation)
```

Comment:
Reading through the sentences, it probably works better to move this one to the end of the paragraph:

> Please refer to the [`antctl` Multi-cluster manual](antctl.md) to learn
more about the kubeconfig file configuration, and the `antctl` Multi-cluster commands. For installation of `antctl`, please refer to the [installation guide](../antctl.md#installation).

What you think?

Reply:
You are right, that looks more readable. I will move it.

Reply:
Updated, thanks.